oxd-server/src/main/java/org/gluu/oxd/server/persistence/PersistenceService.java

```diff
@@ -13,15 +14,31 @@
 
     boolean create(Rp rp);
 
+    boolean createState(String state);
```

Comment:
We need following:
ExpiredObject
```
public class ExpiredObject {
    private String key; 
    private String value;
    private long createdAt;
    private long expiredAt; // createdAt + expirationInterval
    private ExpiredObjectType type; // enum

    // getters/setter
   
}
```

Simple table 
````
CREATE TABLE expired_objects(
    key string,
    value string,
    type string, 
    iat date,
    exp date
);
```

then on PersistenceService
```
    boolean createExpiredObject(ExpiredObject obj);
    ExpiredObject getExpiredObject(String key);
    void deleteExpiredObjects(); // For h2 `delete from expired_object where exp < now()`, for redis nothing
```